commands/init.go

```diff
@@ -37,14 +39,32 @@ func transformInitArgs(args *Args) error {
 		return nil
 	}
 
-	name, err := utils.DirName()
-	if err != nil {
-		return err
+	var (
+		name   string
+		newDir bool
+		err    error
+	)
+
+	if args.IsParamsEmpty() {
+		name, err = utils.DirName()
+		if err != nil {
+			return err
+		}
+	} else {
+		name = args.LastParam()
```

Comment:
Could `LastParam` possibly be an unrelated flag or value for `--template` or similar?